src/qnvimplugin.cpp

```diff
@@ -295,119 +294,119 @@ void QNVimPlugin::syncFromVim() {
-
-                                for (const auto &d : diff) {
-                                    switch (d.command) {
-                                    case Utils::Diff::Insert: {
-                                        qWarning() << 3 << d.text << d.text.size() << d.text.length();
-                                        // Adjust cursor position if we do work in front of the cursor.
-                                        if (charactersInfrontOfCursor > 0) {
-                                            const int size = d.text.size();
-                                            charactersInfrontOfCursor += size;
-                                            newCursorPos += size;
-                                        }
-                                        cursor.insertText(d.text);
-                                        break;
-                                    }
-
-                                    case Utils::Diff::Delete: {
-                                        // Adjust cursor position if we do work in front of the cursor.
-                                        qWarning() << 2 << d.text << d.text.size() << d.text.length();
-                                        if (charactersInfrontOfCursor > 0) {
-                                            const int size = d.text.size();
-                                            charactersInfrontOfCursor -= size;
-                                            newCursorPos -= size;
-                                            // Cursor was inside the deleted text, so adjust the new cursor position
-                                            if (charactersInfrontOfCursor < 0)
-                                                newCursorPos -= charactersInfrontOfCursor;
-                                        }
-                                        cursor.setPosition(cursor.position() + d.text.length(), QTextCursor::KeepAnchor);
-                                        cursor.removeSelectedText();
-                                        break;
-                                    }
-
-                                    case Utils::Diff::Equal:
-                                        // Adjust cursor position
-                                        qWarning() << 1 << d.text << d.text.size() << d.text.length();
-                                        charactersInfrontOfCursor -= d.text.size();
-                                        cursor.setPosition(cursor.position() + d.text.length(), QTextCursor::MoveAnchor);
-                                        break;
-                                    }
-                                }
-                                cursor.endEditBlock();
-                                cursor.setPosition(newCursorPos);
-                            }
+                    case Utils::Diff::Equal:
+                        // Adjust cursor position
+                        qWarning() << 1 << d.text << d.text.size() << d.text.length();
```

Comment:
Not relevant to this PR but I just noticed it while reviewing, it should be safe to delete this line.

Reply:
Does it mean a can delete [this](https://github.com/sassanh/qnvim/pull/42/files/33aae8051e641a65cc2e56372bfd924bbb265f43#diff-2feaaa86ce45da1058ca6c65de08739e349d6b69a3b582ff55d3705eeed712c7R378) and [this](https://github.com/sassanh/qnvim/pull/42/files/33aae8051e641a65cc2e56372bfd924bbb265f43#diff-2feaaa86ce45da1058ca6c65de08739e349d6b69a3b582ff55d3705eeed712c7R365)?

Reply:
Yes, kindly delete them as well.